<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-T3ULBUPM" failures="1" tests="6" name="com.midhun.youtube.testrunner.Runner" time="229.180" errors="0" timestamp="2020-01-28T21:10:12 IST" skipped="0">
  <testcase classname="com.midhun.youtube.testrunner.Runner" name="scenario" time="48.962"/>
  <system-out/>
  <testcase classname="com.midhun.youtube.testrunner.Runner" name="scenario" time="86.312"/>
  <system-out/>
  <testcase classname="com.midhun.youtube.testrunner.Runner" name="scenario" time="47.034">
    <failure message="Result page title mismatch expected [Bakkappa N - TheTechie AutomationLaboratory - YouTube] but found [Selenium by Bakkappa N - YouTube]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Result page title mismatch expected [Bakkappa N - TheTechie AutomationLaboratory - YouTube] but found [Selenium by Bakkappa N - YouTube]
at com.midhun.youtube.util.ExtentReporterNG.testStepHandle(ExtentReporterNG.java:45)
at com.midhun.youtube.util.TestListener.onTestFailure(TestListener.java:30)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:66)
at org.testng.internal.TestInvoker.runTestResultListener(TestInvoker.java:219)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:651)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- scenario -->
  <system-out/>
  <testcase classname="com.midhun.youtube.testrunner.Runner" name="scenario" time="46.626"/>
  <system-out/>
  <testcase classname="com.midhun.youtube.testrunner.Runner" name="scenario" time="0.178"/>
  <system-out/>
  <testcase classname="com.midhun.youtube.testrunner.Runner" name="scenario" time="0.068"/>
  <system-out/>
</testsuite> <!-- com.midhun.youtube.testrunner.Runner -->
